app/src/main/java/app/DTO/Player.kt

```diff
@@ -1,9 +1,18 @@
 package app.DTO
 
+
+/**
+ * The player object to store data.
+ * @param Energy the amount of energy the player has.
+ * @param Money the amount of money the player has.
+ * @param Status the status of the player.
+ * @param Day the day the player is currently on.
+ * @constructor creates a prompt with the given parameters.
+ */
```

Comment:
Added KDOC to this object so that parameters and class can be better understood. more info can be found here: https://kotlinlang.org/docs/kotlin-doc.html#block-tags